astropy/utils/tests/test_data.py

```diff
@@ -446,7 +449,7 @@ def test_get_readable_fileobj_cleans_up_temporary_files(tmpdir, monkeypatch):
     monkeypatch.setattr(tempfile, 'tempdir', str(tmpdir))
 
     # Call get_readable_fileobj() as a context manager
-    with get_readable_fileobj(url) as fileobj:
+    with get_readable_fileobj(url) as fileobj:  # noqa
```

Comment:
not that it's terribly important, but why the noqa here?

Reply:
Because `fileobj` is not used?

Reply:
fileobj isn't used

Reply:
but then why have the ``as`` part?

Reply:
Yep can remove, will do now